.github/workflows/promote.yaml

```diff
@@ -13,8 +13,9 @@ on:
 
 jobs:
   promote:
+    if: contains([1.6, main], github.ref_name)
     name: Promote Charm
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Why not use 22.04?

Reply:
I wanted to make it in sync with the other workflows.
But now I relealised that there is no need to to fix the version in this specific workflow file.
Keeping `ubuntu-latest`.